app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="customer-info">
+  <h1><%= @customer.name %></h1>
+</section>
+
+<section class="item-info">
+  <% @customer.items.each do |item| %>
+    <p><%= "Item Name: #{item.name}" %></p>
```

Comment:
You could wrap these 3 `<p>` tags in a `<div>` and then on that `<div>` element is where you can add the css id that would be something like `#item-<%= item.id %>` so it would be unique for each item.